packages/botonic-core/src/handoff.js

```diff
@@ -151,3 +161,12 @@ export async function getAvailableAgents(session) {
   })
   return resp.data
 }
+
+export function cancelHandoff(session, typification = null) {
+  let params = { typification }
+  session._botonic_action = `discard_case:${JSON.stringify(params)}`
+}
+
+export function deleteUser(session) {
```

Comment:
please add this to botonic-core/index.d.ts:
`export function deleteUser(session: Session): void`